frontend/components/Web/Forms/CreateFishbowl.tsx

```diff
@@ -19,14 +19,15 @@ import { ROUTE_FISHBOWL_DETAIL } from '@/app.config';
 import { locales } from '@/i18n';
 import { useAuth } from '@/contexts/AuthContext';
 import { CREATE_FISHBOWL } from '@/lib/gql/Fishbowl';
-import { formatDateTime } from '@/lib/helpers';
+import { formatDateTime, nearestQuarterHour } from '@/lib/helpers';
 import FormikForm from '@/ui/Form';
 import Input from '@/components/Common/Fields/Input';
 import Textarea from '@/components/Common/Fields/Textarea';
 import Select from '@/components/Common/Fields/Select';
 import DatePicker from '@/components/Common/Fields/DatePicker';
 import SubmitBtn from '@/components/Web/SubmitBtn';
 import FormError from '@/components/Web/Forms/FormError';
+import datePicker from "@/components/Common/Fields/DatePicker";
```

Comment:
Reported as unused by the github builds (we should make this an error instead of warning)

Reply:
@eddsaura We can make a new Issue and approach it later